openmdao_book/other_useful_docs/api_translation.ipynb

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
<p>Also, with this one. Hard to notice the difference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
<p>Added note</p><p> </p>